test/net/sourceforge/kolmafia/utilities/StringUtilitiesTest.java

```diff
@@ -13,8 +15,292 @@ void isVowel() {
+  private static final String HTML_PREFIX = "<html>";
+
+  @Test
+  public void itShouldDoNothingForVariousSimpleInputs() {
+    int testLength = 10;
+    assertNull(StringUtilities.basicTextWrap(null), "Null should neither wrap nor error.");
+    String test = buildTestString(testLength);
+    assertEquals(StringUtilities.basicTextWrap(test), test, "Short input should remain unchanged.");
+    test = HTML_PREFIX + test;
+    assertEquals(
+        test, StringUtilities.basicTextWrap(test), "Short html input should remain unchanged.");
+    testLength = 123;
+    test = buildTestString(testLength);
+    test = HTML_PREFIX + test;
+    assertEquals(
+        test, StringUtilities.basicTextWrap(test), "Long html input should remain unchanged.");
+  }
+
+  @Test
+  public void itShouldWrapForVariousSimpleInputs() {
+    int testLength = 90;
+    String testMe = buildTestString(testLength);
+    String expected = manuallyInsertBreak(testMe, 80);
+    assertEquals(expected, StringUtilities.basicTextWrap(testMe), "Break not where expected.");
+    testLength = 200;
+    testMe = buildTestString(testLength);
+    expected = manuallyInsertBreak(testMe, 80);
+    expected = manuallyInsertBreak(expected, 161);
+    assertEquals(expected, StringUtilities.basicTextWrap(testMe), "Breaks not where expected.");
+  }
+
+  @Test
+  public void itShouldHandleUserInsertedBreaks() {
+    String testMe = buildTestString(159);
+    testMe = manuallyInsertBreak(testMe, 40);
+    String expected = manuallyInsertBreak(testMe, 80);
+    expected = expected + BREAK;
+    assertEquals(expected, StringUtilities.basicTextWrap(testMe), "Breaks not where expected.");
+  }
+
+  @ParameterizedTest
+  @CsvSource({
+    "'1234', '1234'",
+    "'12345', '12345\n'",
+    "'1234567890', '12345\n67890\n'",
+    "'123456789012', '12345\n67890\n12'",
+    "'123\n4567890123456789', '123\n4\n56789\n01234\n56789\n'",
+    "'12 45', '12\n45'",
+    "'1 3 5 7 9 1', '1 3 5\n7 9\n1'",
+    "'1 2  \n  4 5\n 6 \n', '1 2\n4 5\n6'",
+    "'1234 67', '1234\n67'"
+  })
+  public void exerciseTextWrapWithShortWrapLength(String input, String expected) {
+    assertEquals(expected, StringUtilities.basicTextWrap(input, 5));
+  }
+
+  /*
+  This builds a string of the specified length.  It uses the repeating pattern "1234567891" to
+  make it easier for humans to debug.  The modulo 10 is just to get a single digit as a string;
+   */
+  private String buildTestString(int length) {
+    StringBuilder retVal = new StringBuilder();
+    for (int i = 1; i <= length; i++) {
+      retVal.append(i % 10);
+    }
+    return retVal.toString();
+  }
+  /*
+  This inserts a break into a string
+   */
+  private String manuallyInsertBreak(String input, int breakPos) {
+    if (breakPos >= input.length()) {
+      return input;
+    } else {
+      return input.subSequence(0, breakPos) + BREAK + input.subSequence(breakPos, input.length());
+    }
+  }
+  // End tests for basicTestWrap
```

Comment:
you can use `@Nested` to separate tests without having to put comments to break. See e.g. `MaximizerTest.java`